webofneeds/won-owner-webapp/src/main/java/won/owner/pojo/RestStatusResponse.java

```diff
@@ -33,7 +33,10 @@
 
     SETTINGS_CREATED(5200, "Settings created", HttpStatus.CREATED),
 
-    TOS_ACCEPT_SUCCESS(6200, "Successfully accepted Terms Of Service", HttpStatus.OK);
+    TOS_ACCEPT_SUCCESS(6200, "Successfully accepted Terms Of Service", HttpStatus.OK),
+
+    EXPORT_SUCCESS(7200, "Successfully started exporting user", HttpStatus.OK),
+    EXPORT_IS_ANONYMOUS(7403, "To export with an anonymous user you need to provide an email address", HttpStatus.BAD_REQUEST);
```

Comment:
As far as i see these response codes are not pushed into the`won.js`-ResponseCodes yet, we need to add those before we can merge this PR

Reply:
Okay, will do, although js isn't involved in the request here (I found it a bit of a waste to route a simple request through our whole redux system)

Reply:
Response codes are now added to `won.js`